Gemfile.lock

```diff
@@ -1,55 +1,55 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.3.2)
-      actionview (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionpack (6.0.3.3)
+      actionview (= 6.0.3.3)
+      activesupport (= 6.0.3.3)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionview (6.0.3.3)
+      activesupport (= 6.0.3.3)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activerecord (6.0.3.2)
-      activemodel (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activesupport (6.0.3.2)
+    activemodel (6.0.3.3)
+      activesupport (= 6.0.3.3)
+    activerecord (6.0.3.3)
+      activemodel (= 6.0.3.3)
+      activesupport (= 6.0.3.3)
+    activesupport (6.0.3.3)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
       zeitwerk (~> 2.2, >= 2.2.2)
     builder (3.2.4)
-    concurrent-ruby (1.1.6)
+    concurrent-ruby (1.1.7)
     crass (1.0.6)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
     erubi (1.9.0)
-    i18n (1.8.3)
+    i18n (1.8.5)
       concurrent-ruby (~> 1.0)
-    json (2.3.0)
-    jwt (2.2.1)
-    jwt_signed_request (2.5.4)
+    json (2.3.1)
+    jwt (2.2.2)
+    jwt_signed_request (2.6.0)
       jwt (>= 1.5.0)
       rack
-    loofah (2.6.0)
+    loofah (2.7.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     method_source (1.0.0)
     mini_portile2 (2.4.0)
-    minitest (5.14.1)
-    money (6.13.7)
+    minitest (5.14.2)
+    money (6.13.8)
       i18n (>= 0.6.4, <= 2)
-    nokogiri (1.10.9)
+    nokogiri (1.10.10)
```

Comment:
### [nokogiri](https://nokogiri.org)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.10.9 → 1.10.10

[_[change-log](https://nokogiri.org/CHANGELOG.html), [source-code](https://github.com/sparklemotion/nokogiri)_]





<details>
<summary>Commits</summary>

A change of **5** commits. See the full changes on [the compare page](https://github.com/sparklemotion/nokogiri/compare/v1.10.9...v1.10.10).

These are the individual commits:
- (9091602) [ci: only manage the v1.10.x pipeline on this branch](https://github.com/sparklemotion/nokogiri/commit/90916022e38fd48c3a9e3763b0f47f834261d399)
- (c39e1b0) [Support fat binary gems for ruby-2.7](https://github.com/sparklemotion/nokogiri/commit/c39e1b082e73557207db1468e4b567727a384579)
- (f0c324c) [Merge branch '2029-windows-support-for-ruby-27-on-v110x' int…](https://github.com/sparklemotion/nokogiri/commit/f0c324c1d93a2880a32dc46cf8766ff132409806)
- (d2d3c18) [update CHANGELOG for v1.10.10](https://github.com/sparklemotion/nokogiri/commit/d2d3c18a6c2db5b448381573c1248fe480198003)
- (a9a3717) [version bump to v1.10.10](https://github.com/sparklemotion/nokogiri/commit/a9a3717154bdb99ed6d0d785736eb471c9d6a954)


</details>